apps/demo-app/src/app/examples/date-time-picker/disable/disable.component.html

```diff
@@ -0,0 +1,3 @@
+<mat-checkbox (ngModelChange)="setDisabled($event)" [(ngModel)]="disabled">Disabled</mat-checkbox>
+<entry-date-time-picker [formControl]="dateTime" label="Reactive forms"></entry-date-time-picker>
+<entry-date-time-picker [disabled]="disabled" label="Without reactive forms"></entry-date-time-picker>
```

Comment:
It's not possible to set the value of date time picker when we don't use the forms binding (ngModel or formControl)
Hm  in this second example value cannot be set?